lambdas/choice/src/writer.py

```diff
@@ -1,6 +1,89 @@
 import os
+import boto3
+
+"""
+Processes SNS messages containing user choices by writing choices to Dynamo.
+For event and database record shapes, see https://github.com/sjawhar/enchanted-brain/issues/4.
+"""
+
+
+EVENT_USER_ID_KEY = 'userId'
+EVENT_CHOICE_TYPE_KEY = 'choiceType'
+EVENT_CHOICE_KEY = 'choice'
+EVENT_TIMESTAMP_KEY = 'timestamp'
+
+CHOICE_COLOR = 'CHOICE_COLOR'
+CHOICE_EMOTION = 'CHOICE_EMOTION'
+CHOICE_CHILLS = 'CHOICE_CHILLS'
+CHOICE_IMAGERY = 'CHOICE_IMAGERY'
+
+CONCERT_SONG_ID = os.environ['EVENT_NAME']
+DYNAMO_DB_TABLE_NAME = os.environ['DYNAMO_TABLE_NAME']
```

Comment:
Finally, I know this is a really small nitpick and please don't hate me for bringing it up, buuuut....

It's DynamoDB, not Dynamo. `DYNAMODB_TABLE_NAME`

Reply:
No hate! Will fix.